WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -207,6 +207,8 @@ private extension ItemListView {
         static let iconPadding: CGFloat = 26
         static let itemListPadding: CGFloat = 16
         static let bannerCardPadding: CGFloat = 16
+        static let viewHeight: CGFloat = UIScreen.main.bounds.height
+        static let scrollThresholdMultiplier: CGFloat = 1.7
```

Comment:
To clarify, does this multiplier mean the list is scrolled 70% of its total height?